src/@context/Pool/index.tsx

```diff
@@ -191,10 +194,11 @@ function PoolProvider({ children }: { children: ReactNode }): ReactElement {
       !poolData ||
       !poolInfo?.totalPoolTokens ||
       !asset?.chainId ||
-      !accountId
+      !accountId ||
+      !poolInfoUser
```

Comment:
this `poolInfoUser` then also needs o be added to dependency array of this effect

Reply:
no, it will loop then because you set it in that effect. It's a good compromise for now, no time to refactor everything

Reply:
good find! To keep PR reviews sane, could you add a code comment like in this PR please? https://github.com/oceanprotocol/market/pull/1094/files